### ISSUE TITLE
missing App.framework while trying to build - /ios/Flutter/App.framework" failed: No such file or directory     (2)

### ISSUE THREAD

**Issue by danysz** [flutter/flutter#57554]
I update to Flutter **1.17.1** stable channel and since than I am not able to build release version of the app.
The `App.framework` is not available anymore.

I tried this also on the beta channel with version **1.18.0-11.1.pre**

This is the command : `flutter build ios --release --flavor simplesugarbalance --target lib/main-ssb.dart`

Result

<details>
 <summary>log</summary>

```
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:4
    5:24: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
    - (void)contactPicker:(CNContactPickerViewController *)picker didSelectContact:(CNContact *)contact {
                           ^
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:4
    5:1: note: annotate 'contactPicker:didSelectContact:' with an availability attribute to silence this warning
    - (void)contactPicker:(CNContactPickerViewController *)picker didSelectContact:(CNContact *)contact {
    ^
                                                                                                         API_AVAILABLE(ios(9.0))
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:4
    5:81: warning: 'CNContact' is only available on iOS 9.0 or newer [-Wunguarded-availability]
    - (void)contactPicker:(CNContactPickerViewController *)picker didSelectContact:(CNContact *)contact {
                                                                                    ^
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNContact.h:55:12: note: 'CNContact' has been marked as being introduced in iOS 9.0 here, but the deployment target
    is iOS 8.0.0
    @interface CNContact : NSObject <NSCopying, NSMutableCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:4
    5:1: note: annotate 'contactPicker:didSelectContact:' with an availability attribute to silence this warning
    - (void)contactPicker:(CNContactPickerViewController *)picker didSelectContact:(CNContact *)contact {
    ^
                                                                                                         API_AVAILABLE(ios(9.0))
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:1
    22:33: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
    - (void)contactPickerDidCancel:(CNContactPickerViewController *)picker {
                                    ^
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:1
    22:1: note: annotate 'contactPickerDidCancel:' with an availability attribute to silence this warning
    - (void)contactPickerDidCancel:(CNContactPickerViewController *)picker {
    ^
                                                                            API_AVAILABLE(ios(9.0))
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:3
    3:5: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:3
    3:5: note: enclose 'CNContactPickerViewController' in an @available check to silence this warning
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:3
    3:54: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced
    in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:3
    3:54: note: enclose 'CNContactPickerViewController' in an @available check to silence this warning
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    1:19: warning: unused variable 'ims' [-Wunused-variable]
      NSMutableArray *ims = [NSMutableArray array];
                      ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    5:25: warning: 'CNContactFormatter' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      NSString *fullName = [CNContactFormatter stringFromContact:contact
                            ^~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNContactFormatter.h:42:12: note: 'CNContactFormatter' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNContactFormatter : NSFormatter <NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    5:25: note: enclose 'CNContactFormatter' in an @available check to silence this warning
      NSString *fullName = [CNContactFormatter stringFromContact:contact
                            ^~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    6:62: warning: 'CNContactFormatterStyleFullName' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNContactFormatter.h:17:28: note: 'CNContactFormatterStyle' has been marked as being introduced in iOS 9.0 here, but
    the deployment target is iOS 8.0.0
    typedef NS_ENUM(NSInteger, CNContactFormatterStyle)
                               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    6:62: note: enclose 'CNContactFormatterStyleFullName' in an @available check to silence this warning
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    8:30: warning: 'CNContactFormatter' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      NSString * displayName =  [CNContactFormatter stringFromContact:contact
                                 ^~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNContactFormatter.h:42:12: note: 'CNContactFormatter' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNContactFormatter : NSFormatter <NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    8:30: note: enclose 'CNContactFormatter' in an @available check to silence this warning
      NSString * displayName =  [CNContactFormatter stringFromContact:contact
                                 ^~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    9:62: warning: 'CNContactFormatterStyleFullName' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNContactFormatter.h:17:28: note: 'CNContactFormatterStyle' has been marked as being introduced in iOS 9.0 here, but
    the deployment target is iOS 8.0.0
    typedef NS_ENUM(NSInteger, CNContactFormatterStyle)
                               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:5
    9:62: note: enclose 'CNContactFormatterStyleFullName' in an @available check to silence this warning
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    3:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<NSString*> * email in contact.emailAddresses) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    3:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<NSString*> * email in contact.emailAddresses) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    3:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<NSString*> * email in contact.emailAddresses) {
           ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    3:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<NSString*> * email in contact.emailAddresses) {
           ^~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    4:39: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
          [emails addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: email.label ?: @""],
                                          ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    4:39: note: enclose 'CNLabeledValue' in an @available check to silence this warning
          [emails addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: email.label ?: @""],
                                          ^~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
           ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
           ^~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:23: warning: 'CNPhoneNumber' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
                          ^~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNPhoneNumber.h:20:12: note: 'CNPhoneNumber' has been marked as being introduced in iOS 9.0 here, but the deployment
    target is iOS 8.0.0
    @interface CNPhoneNumber : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    8:23: note: enclose 'CNPhoneNumber' in an @available check to silence this warning
      for (CNLabeledValue<CNPhoneNumber*> * phone in contact.phoneNumbers) {
                          ^~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    9:39: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
          [phones addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: phone.label ?: @""],
                                          ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:7
    9:39: note: enclose 'CNLabeledValue' in an @available check to silence this warning
          [phones addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: phone.label ?: @""],
                                          ^~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:8: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
           ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:8: note: enclose 'CNLabeledValue' in an @available check to silence this warning
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
           ^~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:23: warning: 'CNPostalAddress' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
                          ^~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNPostalAddress.h:20:12: note: 'CNPostalAddress' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNPostalAddress : NSObject <NSCopying, NSMutableCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:8
    9:23: note: enclose 'CNPostalAddress' in an @available check to silence this warning
      for (CNLabeledValue<CNPostalAddress*> * address in contact.postalAddresses) {
                          ^~~~~~~~~~~~~~~
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:1
    01:42: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
          [addresses addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: address.label],
                                             ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:6
    :
    In module 'Contacts' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    sUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk/System/Library/Frameworks/Contact
    s.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the
    deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/Shared/develop/flutter/.pub-cache/git/contact_picker-4088389ceee91b263a2cf76bf2a0c65e647c8a2e/ios/Classes/ContactPickerPlugin.m:1
    01:42: note: enclose 'CNLabeledValue' in an @available check to silence this warning
          [addresses addObject:@{ @"label": [CNLabeledValue localizedStringForLabel: address.label],
                                             ^~~~~~~~~~~~~~
    21 warnings generated.
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:25:35: warning:
    'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        fileprivate var allIds: [Int: GADRewardBasedVideoAd] = [:]
                                      ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:93:52: warning:
    'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        private func getRewardBasedVideoAd(id: Int) -> GADRewardBasedVideoAd {
                                                       ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:110:51:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAd(_ rewardBasedVideoAd: GADRewardBasedVideoAd, didRewardUserWith reward: GADAdReward) {
                                                      ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:117:58:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdDidOpen(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                             ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:121:59:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdDidClose(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                              ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:125:61:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdDidReceive(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                                ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:129:66:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdDidStartPlaying(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                                     ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:133:68:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdMetadataDidChange(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                                       ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:137:69:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdDidCompletePlaying(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                                        ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:141:71:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAdWillLeaveApplication(_ rewardBasedVideoAd: GADRewardBasedVideoAd) {
                                                                          ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:145:51:
    warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
        func rewardBasedVideoAd(_ rewardBasedVideoAd: GADRewardBasedVideoAd, didFailToLoadWithError error: Error) {
                                                      ^
    /Users/Shared/develop/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-0.3.4/ios/Classes/AdmobRewardPlugin.swift:95:38: warning:
    'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here:
    https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration.
                let rewardBadedVideoAd = GADRewardBasedVideoAd.sharedInstance()
                                         ^
    building file list ... rsync: link_stat
    **"/Users/danielszasz/projects/digital-cherry/sugar-tracker/sugar_tracker_app/ios/Flutter/App.framework" failed: No such file or directory
    (2)**
    done

    sent 29 bytes  received 20 bytes  98.00 bytes/sec
    total size is 0  speedup is 0.00
    rsync error: some files could not be transferred (code 23) at
    /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-54/rsync/main.c(996) [sender=2.6.9]
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The use of Swift 3 @objc inference in Swift 4 mode is deprecated. Please address deprecated @objc inference warnings, test your
    code with “Use of deprecated Swift 3 @objc inference” logging enabled, and then disable inference by changing the "Swift 3 @objc
    Inference" build setting to "Default" for the "Runner" target. (in target 'Runner' from project 'Runner')
```

</details>

Encountered error while building for device.

This is the result of `flutter doctor -v`

```
[✓] Flutter (Channel stable, v1.17.1, on Mac OS X 10.15.4 19E287, locale en-IL)
    • Flutter version 1.17.1 at /Users/Shared/develop/flutter
    • Framework revision f7a6a7906b (6 days ago), 2020-05-12 18:39:00 -0700
    • Engine revision 6bc433c6b6
    • Dart version 2.8.2

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/Shared/develop/android-sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/Shared/develop/android-sdk
    • Java binary at: /Users/danielszasz/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/192.6392135/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.6)
    • Android Studio at /Users/danielszasz/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/192.6392135/Android
      Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.1.1)
    • IntelliJ at /Users/danielszasz/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 45.1.3
    • Dart plugin version 201.7223.99

[✓] VS Code (version 1.42.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.8.1
```

I found that in the past there was a problem like that but it was solved since I used until few days ago the `beta` channel and it was ok.

**Comment by VladyslavBondarenko** [comment]
could somebody create a minimal project to reproduce the error?

**Comment by jmagman** [comment]
What does your Thin Binary script say?
![Screen Shot 2020-05-19 at 11 15 13 AM](https://user-images.githubusercontent.com/682784/82363028-1d26ab00-99c2-11ea-9c1c-43432c4ad36d.png)

May be related to the migration to fix https://github.com/flutter/flutter/issues/50568
https://flutter.dev/docs/development/ios-project-migration

**Comment by jmagman** [comment]
> I found that in the past there was a problem like that but it was solved since I used until few days ago the `beta` channel and it was ok.

You are saying it was working a few days ago in the beta channel and now it isn't?  Do you remember what version it was working in?

**Comment by danysz** [comment]
yes it was working with the latest version beta which was relevant before
the stable 1.17.1 was released. if i remember well it was some 1.19 but i
don't remember for sure

**Comment by jmagman** [comment]
@danysz What does your Thin Binary script say?

**Comment by danysz** [comment]
> @danysz What does your Thin Binary script say?

/bin/sh "$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh" embed_and_thin

**Comment by jmagman** [comment]
Hm that looks right...

Can you attach the full output of `flutter build ios --release --verbose --flavor simplesugarbalance --target lib/main-ssb.dart` (it'll be large).

**Comment by danysz** [comment]
> Hm that looks right...
> 
> Can you attach the full output of `flutter build ios --release --verbose --flavor simplesugarbalance --target lib/main-ssb.dart` (it'll be large).

[log-build.txt.zip](https://github.com/flutter/flutter/files/4652473/log-build.txt.zip)

**Comment by jmagman** [comment]
I don't see the Run Script Xcode build phase that actually calls back into Flutter to compile your dart code.

Can you open `ios/Runner.xcworkspace` and go to your Runner build phases and give us a screen shot?

It should look like:
![Screen Shot 2020-05-19 at 11 57 11 AM](https://user-images.githubusercontent.com/682784/82367094-1dc24000-99c8-11ea-8f5b-9c09bce28d0e.png)

Is it possible it was removed when you added "Copy GoogleService-Info.plist to correct location"?

**Comment by danysz** [comment]
> I don't see the Run Script Xcode build phase that actually calls back into Flutter to compile your dart code.
> 
> Can you open `ios/Runner.xcworkspace` and go to your Runner build phases and give us a screen shot?
> 
> It should look like:
> ![Screen Shot 2020-05-19 at 11 57 11 AM](https://user-images.githubusercontent.com/682784/82367094-1dc24000-99c8-11ea-8f5b-9c09bce28d0e.png)
> 
> Is it possible it was removed when you added "Copy GoogleService-Info.plist to correct location"?

Amazing...you were right. Added the line and now is working.

I think that it will be really nice to have a check about missing "stuff" from the process or to put it in a way it can't be deleted.

**Comment by jmagman** [comment]
> I think that it will be really nice to have a check about missing "stuff" from the process or to put it in a way it can't be deleted.

Unfortunately those checks are in that script, and the script was deleted. 😄 

We can't prevent you from opening your Xcode project and doing whatever you want in there since there are many legit reasons to edit your Xcode project (adding flavors, changing your signing details, adding watchOS or rich notifications support, etc).

I doubt the rest 6 of who 👍 this issue accidentally manually deleted this build phase.  Can you create new GitHub issues, completely fill out the template, and include a `--verbose` output of your `flutter run`, as well as attaching `ios/Runner.xcodeproj/project.pbxproj`?

**Comment by marekkxyz** [comment]
As you can see I have filled `Run Script` filed and have the same issue - no App.framework building :/
<img width="719" alt="Screenshot 2020-05-20 at 08 41 15" src="https://user-images.githubusercontent.com/30255564/82413663-0c377180-9a76-11ea-9165-3de0a25f9df0.png">

**Comment by jmagman** [comment]
@marekkxyz Did you add those input files?  That's not right, the script won't run unless your CocoaPod dependencies change...  The `flutter create` template doesn't have that.

**Comment by jmagman** [comment]
I'm going to close this.  Flutter is generating an Xcode project for you that has the scripts set up to compile and embed your dart code, and embed the Flutter framework.  If you delete the script or change the dependency inputs, that script it won't run.  There's no way to prevent you from doing this, at least without very large, fundamental changes to the `flutter` tool that, say, regenerates the Xcode project on every run and applies your bundle identifier and display name and signing identifier and anything else legit you added there (or something like that).

Always use source control so you can easily revert manual changes you make to your app or at least bisect to see where it started breaking.  If that doesn't work, you can always let Flutter regenerate the iOS project:
```
rm -rf ios/
flutter create .
```
Then reapply [any changes you need to make in order to ship your app](https://flutter.dev/docs/deployment/ios).

If, however, you didn't make any manual changes like deleting the script out from under Flutter, please file a GitHub issue and let us know, and completely fill out the template.  Give us as many logs and reproduction steps as possible while you are reproducing it.  When you apply random workarounds you find on Stack Overflow before you report the Flutter bug, it's really hard for us to track them down and fix to make the experience better for everyone.

For the other 5 of you who 👍 this issue: if you didn't manually edit the Flutter build script phase in Xcode, please file a new GitHub issue so we can investigate.  Include:
1. `flutter doctor -v`
2. The `flutter run` command with the addition of the `--verbose` flag (it's large, you can attach it instead of pasting it)
3. ` ios/Runner.xcodeproj/project.pbxproj`

**Comment by joniaiuser** [comment]
> I'm going to close this. Flutter is generating an Xcode project for you that has the scripts set up to compile and embed your dart code, and embed the Flutter framework. If you delete the script or change the dependency inputs, that script it won't run. There's no way to prevent you from doing this, at least without very large, fundamental changes to the `flutter` tool that, say, regenerates the Xcode project on every run and applies your bundle identifier and display name and signing identifier and anything else legit you added there (or something like that).
> 
> Always use source control so you can easily revert manual changes you make to your app or at least bisect to see where it started breaking. If that doesn't work, you can always let Flutter regenerate the iOS project:
> 
> ```
> rm -rf ios/
> flutter create .
> ```
> 
> Then reapply [any changes you need to make in order to ship your app](https://flutter.dev/docs/deployment/ios).
> 
> If, however, you didn't make any manual changes like deleting the script out from under Flutter, please file a GitHub issue and let us know, and completely fill out the template. Give us as many logs and reproduction steps as possible while you are reproducing it. When you apply random workarounds you find on Stack Overflow before you report the Flutter bug, it's really hard for us to track them down and fix to make the experience better for everyone.
> 
> For the other 5 of you who 👍 this issue: if you didn't manually edit the Flutter build script phase in Xcode, please file a new GitHub issue so we can investigate. Include:
> 
> 1. `flutter doctor -v`
> 2. The `flutter run` command with the addition of the `--verbose` flag (it's large, you can attach it instead of pasting it)
> 3. ` ios/Runner.xcodeproj/project.pbxproj`

Just to add on to this, I recently experienced this as well and managed to fix it by opening Xcode, going to Runner > Build Phases and manually dragging-and-dropping Thin Binary to the last step of the build process. For some reason unknown, 1.17.x upgrade or something else may have reordered some of the build scripts.. Once I did that I was able to flutter clean and build successfully without this issue. Hope it helps.
